frontend/src/index.jsx

```diff
@@ -29,18 +34,36 @@ function App() {
 
 	return (
 		<div>
-			<NavBar />
-			{/* <LandingPage /> */}
-			{/* currently swapped for testming, swap back before commiting */}
-			{isMobile ? <LandingPageMobile /> : <LandingPage />}
-			{/* re-add if timer is fixed for both mobile and web */}
-			{/* <Timer /> */}
-			<AboutPage />
-			<FAQPage />
-			<SponsorPage/>
-			<Footer />
+			<Routes>
+				<Route path="/" element={<Layout />}>
+					{isMobile ?
+						<Route index element={<LandingPageMobile />} />
+							: <Route index element={<LandingPage />} />
+					}
+					{/* Landing Pages are the default route (as they should be) */}
+
+					<Route path="about" element={<AboutPage />} />
+					<Route path="faq" element={<FAQPage />} />
+					<Route path="sponsors" element={<SponsorPage />} />
+
+
+					<Route path="*" element={<ErrorPage />} />
```

Comment:
This doesn't seem to be working properly, if I enter a junk route the page displays "Cannot GET /junk-route"